ddtrace/context.py

```diff
@@ -55,6 +59,15 @@ def close_span(self, span):
             self._finished_spans += 1
             self._current_span = span._parent
 
+            # notify if the trace is not closed properly
+            tracer = getattr(span, '_tracer', None)
+            if tracer and tracer.debug_logging and span._parent is None:
+                opened_spans = len(self._trace) - self._finished_spans
```

Comment:
yea it's a useless information in that case, we can prevent logging anything if all it's good. Thank you!